python/ambassador/envoy/v3/v3_static_resources.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, config: "V3Config") -> None:
 
         self.update(
             {
-                "listeners": [l.as_dict() for l in config.listeners],
+                "listeners": [x.as_dict() for x in config.listeners],
```

Comment:
```suggestion
                "listeners": [listener.as_dict() for listener in config.listeners],
```

Absolutely a tiny nit, please feel 100% free to ignore it, just a personal preference for short-but-descriptive variable names over generic or shortest-possible ones.